lib/dialects/postgres/query-generator.js

```diff
@@ -412,14 +412,15 @@ class PostgresQueryGenerator extends AbstractQueryGenerator {
     ].join(' ');
   }
 
-  removeIndexQuery(tableName, indexNameOrAttributes) {
+  removeIndexQuery(tableName, indexNameOrAttributes, options) {
+    const concurrently = options && options.concurrently ? ' CONCURRENTLY ' : ' ';
     let indexName = indexNameOrAttributes;
 
     if (typeof indexName !== 'string') {
       indexName = Utils.underscore(`${tableName}_${indexNameOrAttributes.join('_')}`);
     }
 
-    return `DROP INDEX IF EXISTS ${this.quoteIdentifiers(indexName)}`;
+    return `DROP INDEX${concurrently}IF EXISTS ${this.quoteIdentifiers(indexName)}`;
```

Comment:
is this more readible?

```
return [
  "DROP INDEX",
  options && options.concurrently && "CONCURRENTLY",
  `IF EXISTS ${this.quoteIdentifiers(indexName)}`
].filter(Boolean).join(' ');
```

Reply:
Definitely! Makes sense, i noticed this pattern in other places too. Updated and also rebased.